README.md

```diff
@@ -28,3 +28,12 @@ When adding to an existing instance, you'll have to create the new SQL-tables us
 ```bash
 invenio alembic upgrade
 ```
+
+## `invenio-jobs` integration
+
+`invenio-edugain`'s idp-data ingestion can be run via the _jobs_ view in the administration UI.
+Note that the _jobs_ view is hidden by default, enable it via adding the following configuration:
```

Comment:
should we link here to the documentation how to configure it InvenioRDM?  i mean i wouldn't understand with this documentation to add it to `invenio.cfg`

Reply:
`grep`ing through the code for [docs](https://github.com/inveniosoftware/docs-invenio-rdm) reveals no documentation for the `jobs` feature yet

Added a TODO to link documenation once it was written